apps/arkmarket/src/types/index.ts

```diff
@@ -120,11 +120,11 @@ export interface PortfolioToken {
 
 export interface TokenOffer {
   expire_at: number;
-  floor_difference: number | null;
+  floor_difference?: number;
```

Comment:
Can `floor_difference` be `undefined` when returned from the API? I think that they use `null` instead